pycontrails/core/met.py

```diff
@@ -280,7 +283,7 @@ def _preprocess_dims(self, wrap_longitude: bool) -> None:
             # Only shift if necessary
             if np.any(lon >= 180.0) or np.any(lon < -180.0):
                 self.data = shift_longitude(self.data)
-            else:
+            elif sort:
```

Comment:
We already call `self.data.sortby(...)` 11 lines above this. Why make this sort conditional? There is also a `copy` parameter in the `MetDataset` constructor. If that is set to False, then I think we avoid unnecessary sorting.